src/app.js

```diff
@@ -45,38 +32,11 @@ export async function build(opts = {}) {
             }
         })
 
-    // the body will look like:  {credentialId: '23kdr', credentialStatus: [{type: 'StatusList2021Credential', status: 'revoked'}]}
-    app.post("/instance/:instanceId/credentials/status", 
-        async (req, res) => {
-            try {
-                if (!enableStatusAllocation) return res.status(405).send("The status service has not been enabled.")
-                const instanceId = req.params.instanceId 
-                try {
-                    await verifyAccess(req.headers.authorization, instanceId)
-                } catch (e) {
-                    return res.status(e.code).send(e.message)
-                }
-                if (!req.body || !Object.keys(req.body).length ) return res.status(400).send('No update request was provided in the body')
-                
-                const {credentialId, credentialStatus} = req.body;
-                const status = credentialStatus[0].status
-                const statusType = credentialStatus[0].type
-                if (statusType === 'StatusList2021Credential') {
-                    const statusResponse = await revoke(credentialId, status)
-                    return res.status(statusResponse.code).send(statusResponse.message)
-                } else {
-                    return res.status(400).send('StatusList2021Credential is the only supported revocation mechanism.')
-                }
-            } catch (error) {
-                console.log(error);
-                return res.status(500).json(error);
-            }
-        })
-
-        app.get('/seedgen', async (req, res, next) => {
-            const newSeed = await generateSeed()
-            res.send(newSeed)
-        });
+    app.get('/seedgen', async (req, res, next) => {
```

Comment:
what's this one for?

Reply:
This is a convenience endpoint that will generate a new seed, as explained here: https://github.com/digitalcredentials/signing-service/blob/8ae3e5e0b2a919c90bade0eb6d9c5c91df2c5f63/README.md#signing-key

It just generates a new random seed and returns it, along with the associated DID:KEY and DID document.